server/src/routes/course/crossCheck/createMessage.ts

```diff
@@ -67,9 +68,15 @@ export const createMessage = (_: ILogger) => async (ctx: Router.RouterContext) =
     user: user,
   });
 
+  const userId = await getUserId(student.id, taskSolutionResult.checkerId, inputData.role);
+  if (!userId) {
+    setErrorResponse(ctx, StatusCodes.BAD_REQUEST, 'user not found');
```

Comment:
how about `NOT_FOUND` status code?

Reply:
IMO this is a bad request as we can't process current action due to missing data. It might be not found if we couldn't find cross check result for example.